### PR TITLE
Inline CTAs For Fullscreen Marquee

### PR DESCRIPTION
**Pull request by echen-adobe**
Describe your specific features or fixes

Transforms CTAs in fullscreen-marquee to using display:inline-block so that they float next to each other. This allows multiple CTAs to be supported.

Resolves: [MWPW-NUMBER](https://jira.corp.adobe.com/browse/MWPW-152925)

Test URLs:
- Before: https://main--express--adobecom.hlx.page/express/
- After: https://fullscreen-marquee-cta--express--adobecom.hlx.page/express/create/flyer

Screenshot 2024-06-25 at 11 06 15 AM

### CODE REVIEW

File: express/blocks/fullscreen-marquee/fullscreen-marquee.css [hunk 1]
```diff
@@ -47,8 +40,10 @@
 .fullscreen-marquee .fullscreen-marquee-heading .button-container {
   align-items: center;
   display: none;
+  ;
```
Comment (qiyundai) [inline express/blocks/fullscreen-marquee/fullscreen-marquee.css:43]: Let's keep it tidy.

**Review by kennethlum — APPROVED** [review]

File: express/blocks/fullscreen-marquee/fullscreen-marquee.css (hunk 1 above)
Comment (echen-adobe) [reply express/blocks/fullscreen-marquee/fullscreen-marquee.css]: Sorry about that, I thought the linter would take care of it.

File: express/blocks/fullscreen-marquee/fullscreen-marquee.css [hunk 2]
```diff
@@ -277,8 +275,12 @@
     text-align: center;
   }
 
+  .fullscreen-marquee .fullscreen-marquee-heading p:first-of-type {
+    margin: 10px auto 0px auto;
+  }
+
   .fullscreen-marquee .fullscreen-marquee-heading .button-container {
-    display: flex;
+    display: inline-block;
```
[the remark this answers is not in the document]
Comment (echen-adobe) [reply express/blocks/fullscreen-marquee/fullscreen-marquee.css:282]: Not seeing the need for it unless the button container contains multiple children. I think it only contains a single button for this block.

**Review by JingleH — APPROVED** [review]

File: express/blocks/fullscreen-marquee/fullscreen-marquee.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (kennethlum) [reply express/blocks/fullscreen-marquee/fullscreen-marquee.css:282]: `inline-flex` can be more versatile, but if `inline-block` already fits the purpose, then go ahead

**Pull request merged**